### PR TITLE
[2019-TLV] Opened Registration and Added Speakers

### PR DESCRIPTION
**Pull request by shar1z**
I managed to get Hugo running locally - so it seems that everything is in order to merge.  Thanks!

### CODE REVIEW

File: data/events/2019-tel-aviv.yml [hunk 1]
```diff
@@ -180,7 +183,7 @@ sponsors:
   - id: devopsloft
     level: community
 
-sponsors_accepted : "yes" # Whether you want "Become a XXX Sponsor!" link
+sponsors_accepted : "" # Whether you want "Become a XXX Sponsor!" link
```
Comment (mattstratton) [inline data/events/2019-tel-aviv.yml]: Is the intent to close sponsorship? As it happens, i don’t think this part of the code works anyway (the “join as XXX sponsor” links still show up anyway)

**Comment by mattstratton** [comment]
@shar1z - this should be okay to merge, but i had one question first (see above). If that’s okay, or even if you want us to merge and look into that later, just let us know in the PR here and we will merge right away!

File: data/events/2019-tel-aviv.yml (hunk 1 above)
Comment (shar1z) [reply data/events/2019-tel-aviv.yml]: Yes I technically want to close sponsorships - as we're sold out.  I don't mind leaving it "yes" if it doesn't work anyway.  I'd like to remove the sponsor links though.  I'll try and do that now. 

Thanks!

**Comment by bridgetkromhout** [comment]
@shar1z you can remove all sponsorship links if desired - I wasn't sure if that's what you were trying to do?

**Pull request merged**